product_docs/docs/pgd/5/node_management/maintainance_with_proxies.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: Maintenance commands through proxies
+---
+
+## Maintenance and performance
+
+As a general rule, you should never perform maintenance operations on a cluster's write leader. 
+Maintenance operations such as `VACUUM` can be quite disruptive to the smooth running of a busy server and often detrimental to workload performance.
+Therefore it is best to run maintenance commands on any node in a group that isn't the write leader.
+Generally, this requires you connect directly and issue the maintenance commands on the non-write leader nodes.
+But there are situations where this is not possible.
+
+## Maintenance and proxies
+
+Proxies, by design, always connect to and send commands to the current write leader.
+This would usually mean that you should not use a proxy for maintenance.
```

Comment:
"...should not connect via a proxy to perform maintenance"

Reply:
Fixed